plugin/documents.py

```diff
@@ -526,7 +526,7 @@ def _apply_view_settings(self, session: Session) -> None:
         if any(trigger.get('server', None) == session.config.name for trigger in completion_triggers):
             return
         # This is to make ST match with labels that have a weird prefix like a space character.
-        settings.set('auto_complete_preserve_order', 'none')
+        # settings.set('auto_complete_preserve_order', 'none')
```

Comment:
Hmm?

Reply:
Whoops :) I was playing around with this yesterday.